.circleci/config.yml

```diff
@@ -5,25 +5,12 @@ aliases:
     name: Install Podman in Ubuntu Focal
     command: ./install/linux/install-podman-ubuntu-focal.sh
 
-    # FIXME: Remove the following step once we drop Ubuntu Focal support. The
-    # python-all dependency is an artificial requirement due to an stdeb bug
-    # prior to v0.9.1. See:
-    #
-    # * https://github.com/astraw/stdeb/issues/153
-    # * https://github.com/freedomofpress/dangerzone/issues/292#issuecomment-1349967888
-  - &install-python-all
-    name: Install python-all package
-    command: |
-      export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true
-      apt-get update
-      apt-get install -y python-all
-
   - &install-dependencies-deb
     name: Install dependencies (deb)
     command: |
       export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true
       apt-get update
-      apt-get install -y dh-python python3 python3-stdeb
+      apt-get install -y dh-python python3 python3-setuptools dpkg-dev debhelper
```

Comment:
If you're in the DZ checkout, you can just run `apt build-dep .` instead of needing to duplicate the Build-Requires here.

Reply:
Thanks! Just did.